### DISCUSSION TITLE
소셜 로그인 유저의 게스트 댓글 조작여부

### DISCUSSION THREAD

**Discussion by zereight** [darass-team/2021-darass#728] [Ideas]
현재 소셜로그인 유저는 게스트댓글을 조작할 수 없도록 해놨는데, (관리자는 게스트 댓글의 삭제는 가능함)

소셜로그인 유저가 
(비밀번호를 맞추면) 게스트 댓글의 조회/수정/삭제가 가능하게 하는것이 좋을지?

**Comment by pjy1368** [comment]
기본적으로 비밀 댓글의 목적인 작성자와 괸리자만 보는 것이라 생각해서, 나는 소셜 로그인 유저는 남의 비밀 댓글은 못보는게 맞다고 생각해.

**Reply by zereight** [reply]
의견 감사합니다~

**Comment by jujubebat** [comment]
저도 소셜 로그인 유저는 자신의 댓글만 조작하는게 좋다고 생각합니다!

**Reply by zereight** [reply]
의견 감사합니다~
